common/Environments/Helpers/ComputeSystemHelpers.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices.WindowsRuntime;
+using System.Text;
+using System.Threading.Tasks;
+using DevHome.Common.Environments.Models;
+using DevHome.Common.Helpers;
+using Microsoft.UI.Xaml.Media.Imaging;
+using Microsoft.Windows.DevHome.SDK;
+
+namespace DevHome.Common.Environments.Helpers;
+
+public static class ComputeSystemHelpers
+{
+    public static async Task<BitmapImage?> GetBitmapImageAsync(ComputeSystem computeSystemWrapper)
+    {
+        try
+        {
+            var result = await computeSystemWrapper.GetComputeSystemThumbnailAsync(string.Empty);
+
+            if (result.Result.Status == ProviderOperationStatus.Failure || result.ThumbnailInBytes.Length <= 0)
```

Comment:
Coding guidelines: Use parenthesis. We have for C++ in Windows and it's also in "[Common C# code conventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions#:~:text=Use%20parentheses%20to%20make%20clauses%20in%20an%20expression%20apparent%2C%20as%20shown%20in%20the%20following%20code.)"